lib/knexFunctions.js

```diff
@@ -14,19 +15,133 @@ function createGame(id){
   })
   .returning("id")
   .then(function(gameid){
-    allGames[gameid] = (gops.createGameObject("gops", id));
+    return knex('user_games')
+    .insert({
+      user_id: userid,
+      game_id: gameid[0]
+    }).returning("game_id")
+  }).then(function(gid){
+    allGames[gid[0]] = (gops.createGameObject("gops", userid));
+    console.log(allGames);
   })
 }
 
-//gameID from params, playerID from cookies.
+//TO TRIGGER: Player clicks "Join" from server list
+//Using gameID from URL and userID from cookies, creates a player object in the appropriate game.
 function addPlayer(gameid, userid) {
 
   gops.appendPlayerToGame(allGames[gameid], userid);
+
+  knex('user_games')
+  .insert({
+    user_id: userid,
+    game_id: gameid
+  })
+  .then(function(){
+    console.log("knex write successful")
+  })
+};
+
+//TO TRIGGER: Number of players in game = max number to start game.
+// Sets status of game to "Active", removing it from gamelist.
+function startGame(gameid) {
+  if(gops.startCheck(allGames[gameid])) {
+
+    knex('gamedetails')
+    .where('game_id', '=', gameid)
+    .update({
+      status: "Active"
+    })
+  }
 };
 
-//GameID and card will come from Params
+//TO TRIGGER: Player submits a card
+//Sets selected card as readyCard for player.
 function playHand(gameid, userid, card) {
 
   allGames[gameid][userid].readyCard = card;
 }
 
+//TO TRIGGER: Should run after each player submits a card,
+//System will run a check to see if everyone has submitted a card. If true, decides round winner and award points.
+function winRound(gameid) {
```

Comment:
hmm. Yes, however it's not just checking, it's also appointing the winner if the check returns true.